src/webhook/validation/deprecation.go

```diff
@@ -17,7 +17,7 @@ func deprecatedFeatureFlagFormat(dv *dynakubeValidator, dynakube *dynatracev1bet
 
 	deprecatedPrefix := dynatracev1beta1.DeprecatedFeatureFlagPrefix + dynatracev1beta1.FeatureFlagAnnotationBase
 	if len(dynatracev1beta1.FlagsWithPrefix(dynakube, deprecatedPrefix)) > 0 {
-		return fmt.Sprintf(featureDeprecatedWarningMessage, "'alpha.' prefix not necessary")
+		return fmt.Sprintf(featureDeprecatedWarningMessage, "'alpha.' prefix not necessary for dynakube feature-flags")
```

Comment:
would give them a more precise instruction on what this means for them. Like `alpha. prefix can be removed from feature-flags`